
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 09:19:46,912                                   
【执行时间】[69284]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.IO.InvalidDataException: Multipart body length limit 16384 exceeded.
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.UpdatePosition(Int32 read)
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.ReadAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.StreamHelperExtensions.DrainAsync(Stream stream, ArrayPool`1 bytePool, Nullable`1 limit, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.MultipartReader.ReadNextSectionAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.InnerReadFormAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.ReadForm()
   at Microsoft.AspNetCore.Http.DefaultHttpRequest.get_Form()
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 31
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 09:23:35,801                                   
【执行时间】[87297]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.IO.InvalidDataException: Multipart body length limit 16384 exceeded.
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.UpdatePosition(Int32 read)
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.ReadAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.StreamHelperExtensions.DrainAsync(Stream stream, ArrayPool`1 bytePool, Nullable`1 limit, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.MultipartReader.ReadNextSectionAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.InnerReadFormAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.ReadForm()
   at Microsoft.AspNetCore.Http.DefaultHttpRequest.get_Form()
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 31
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 09:23:37,788                                   
【执行时间】[89284]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.IO.InvalidDataException: Multipart body length limit 16384 exceeded.
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.UpdatePosition(Int32 read)
   at Microsoft.AspNetCore.WebUtilities.MultipartReaderStream.ReadAsync(Byte[] buffer, Int32 offset, Int32 count, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.StreamHelperExtensions.DrainAsync(Stream stream, ArrayPool`1 bytePool, Nullable`1 limit, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.WebUtilities.MultipartReader.ReadNextSectionAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.InnerReadFormAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Http.Features.FormFeature.ReadForm()
   at Microsoft.AspNetCore.Http.DefaultHttpRequest.get_Form()
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 31
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 09:47:48,692                                   
【执行时间】[29011]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.NullReferenceException: Object reference not set to an instance of an object.
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 28
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 10:08:38,415                                   
【执行时间】[302797]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】出现未知异常                                   
【错误详情】
System.NullReferenceException: Object reference not set to an instance of an object.
   at WeChat.Core.Api.Controllers.ImageController.Delete(String key)
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 10:21:58,916                                   
【执行时间】[33761]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.IO.IOException: 文件名、目录名或卷标语法不正确。 : 'F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\form-data; name="files"; filename="DSC02203.jpg.JPG"'
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
   at System.IO.StreamReader..ctor(String path)
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 38
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 17:56:37,912                                   
【执行时间】[42146]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.NullReferenceException: Object reference not set to an instance of an object.
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 33
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 17:57:53,067                                   
【执行时间】[43137]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.NullReferenceException: Object reference not set to an instance of an object.
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 33
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-01-22 17:58:06,990                                   
【执行时间】[57061]毫秒                                   
【错误位置】System.String 属性[(null)]                                   
【错误描述】System.NullReferenceException: Object reference not set to an instance of an object.
   at WeChat.Core.Api.Middleware.LogReqResponseMiddleware.Invoke(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\LogReqResponseMiddleware.cs:line 33
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WeChat.Core.Api.Middleware.CustomExceptionMiddleware.InvokeAsync(HttpContext httpContext) in F:\ERP_Code\WeChat.Core.Api\src\WeChat.Core.Api\Middleware\CustomExceptionMiddleware.cs:line 24                                   
【错误详情】
